packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -97,7 +97,15 @@ export type GPCPCDArgs = {
    */
   id?: StringArgument;
   // TODO(POD-P3): Support PODValue of multiple types.
-};
+} /**
+ * POD objects to prove about. Each object is identified by name and prefixed
+ * with {@link PODPCD_ARG_PREFIX} followed by an underscore.  These are not
```

Comment:
I think for external developers we should say `podpcd` and expect them to use that prefix as-is rather than the constant (you could link to the constant in a parenthetical so devs know it's available).  I think we're unlikely to feel we can safely change this prefix once it's in use (and we have any compatibility guarantees), so given that I'd suggest erring on the side of simplicity and clarity rather than maximizing generality.